test/acceptance/behave/components/ngsiv2/entities/create_entities.feature

```diff
@@ -0,0 +1,1899 @@
+  Scenario:  try to create entities in NGSI v2 without content-type header
+    Given  a definition of headers
+      | parameter          | value             |
+      | Fiware-Service     | test_content_type |
+      | Fiware-ServicePath | /test             |
+    When create "1" entities with "1" attributes
+      | parameter        | value       |
+      | entities_type    | room        |
+      | entities_id      | room2       |
+      | attributes_name  | temperature |
+      | attributes_value | 34          |
+    Then verify that receive several "Unsupported Media Type" http code
+    And verify several error responses
+      | parameter   | value                                                                           |
+      | error       | Unsupported Media Type                                                          |
+      | description | Content-Type header not used, default application/octet-stream is not supported |
+
+  @content_type_error
+  Scenario Outline:  try to create entities in NGSI v2 with wrong content-type header
+    Given  a definition of headers
+      | parameter          | value             |
+      | Fiware-Service     | test_content_type |
+      | Fiware-ServicePath | /test             |
+      | Content-Type       | <content_type>    |
+    When create "1" entities with "1" attributes
+      | parameter        | value       |
+      | entities_type    | room        |
+      | entities_id      | room2       |
+      | attributes_name  | temperature |
+      | attributes_value | 34          |
+    Then verify that receive several "Unsupported Media Type" http code
+    And verify several error responses
+      | parameter   | value                                      |
+      | error       | Unsupported Media Type                     |
+      | description | not supported content type: <content_type> |
+    Examples:
+      | content_type                      |
+      | application/x-www-form-urlencoded |
+      | application/xml                   |
+      | multipart/form-data               |
+      | text/plain                        |
+      | text/html                         |
+      | dsfsdfsdf                         |
+      | <sdsd>                            |
+      | (eeqweqwe)                        |
+
+  # ---------- Services --------------------------------
+  @service_without
+  Scenario: create entities in NGSI v2 without service header
+    Given  a definition of headers
+      | parameter          | value            |
+      | Fiware-ServicePath | /test            |
+      | Content-Type       | application/json |
+    When create "1" entities with "1" attributes
+      | parameter        | value       |
+      | entities_type    | room        |
+      | entities_id      | room2       |
+      | attributes_name  | temperature |
+      | attributes_value | 34          |
+    Then verify that receive several "Created" http code
+    And verify that entities are stored in mongo
```

Comment:
Not sure if I'm interpreting correctly the semantics in the test steps, but probably it could be more precise to use things like this:

"And verify that entities are stored in mongo in the XXX tenant"

or

And verify that entities are stored in mongo in the default tenant"